cmd/kube-apiserver/app/server.go

```diff
@@ -244,15 +244,13 @@ func CreateNodeDialer(s completedServerRunOptions) (tunneler.Tunneler, *http.Tra
 		if s.KubeletConfig.Port == 0 {
 			return nil, nil, fmt.Errorf("must enable kubelet port if proxy ssh-tunneling is specified")
 		}
-		if s.KubeletConfig.ReadOnlyPort == 0 {
-			return nil, nil, fmt.Errorf("must enable kubelet readonly port if proxy ssh-tunneling is specified")
```

Comment:
Is the removal of "ReadOnlyPort" going to break ssh-tunneling?

Reply:
It should not, the ssh-tunneling uses port 22 for the tunnel and whatever is specified for the healthz port is only used for doing a healthz check, the ReadOnlyPort shouldn't be used anywhere in that process after this change.